.gitignore

```diff
@@ -39,6 +39,8 @@ i18n/*.qm
 Makefile
 *.pro.user
 *.stash
+.ccache/
+.ci/
```

Comment:
actually .ci cannot be ignored, it's part of the QGIS ci configuration

Reply:
ack.